<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="str1_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>str1</string>
   </property>
  </widget>
  <widget class="QLabel" name="str2_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>str2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>200</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>数字</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>121</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="count">
       <property name="text">
        <string>count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="size">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="indexof">
       <property name="text">
        <string>indexof</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lastindexof">
       <property name="text">
        <string>lastIndexof</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>字符串</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>134</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QToolButton" name="right">
       <property name="text">
        <string>right</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="lower">
       <property name="text">
        <string>toLower</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="prepend">
       <property name="text">
        <string>Prepend</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="left">
       <property name="text">
        <string>left</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="append">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="upper">
       <property name="text">
        <string>toUpper</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="section">
       <property name="text">
        <string>section</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="trimm">
       <property name="text">
        <string>trimmed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>逻辑判断</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="str1_editText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="str2_editText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结果</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="result_editText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>240</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QToolButton" name="is_null">
      <property name="text">
       <string>isNull</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="contains">
      <property name="text">
       <string>contains</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="endwith">
      <property name="text">
       <string>endsWith</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="stratswith">
      <property name="text">
       <string>startsWith</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="is_empty">
      <property name="text">
       <string>isEmpty</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="count_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>count</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="count_editText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_count_right_or_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>截断字符数量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editText_count_right_or_left">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
